client/signup/steps/site-type/index.jsx

```diff
@@ -15,6 +15,12 @@ import StepWrapper from 'signup/step-wrapper';
 import { getSiteType } from 'state/signup/steps/site-type/selectors';
 import { submitSiteType } from 'state/signup/steps/site-type/actions';
 
+const siteTypeToFlowname = {
+	'online-store': 'ecommerce-onboarding',
+	business: 'onboarding-for-business',
```

Comment:
Since we're adding the preview to the `onboarding` flow, do we still need to fork to `onboarding-for-business` or can it be removed?

Reply:
Looks like it could be safely removed - all the steps would be the same

Reply:
Nice! I wasn't sure if there was still some significance in the business flow - Shall I update so that there's just a blog specific flow without the preview, or move on with all segments going through the regular onboarding (with preview) flow, ready for when we're unblocked on the blog segment?

Reply:
I think a blog flow without the preview and then business and professional going through Onboarding is the best scenario so that we can launch Professional when it's ready and not be held up by the Blog segment.

Reply:
> Looks like it could be safely removed - all the steps would be the same

Ah, okay. Great to know. I was confused about whether we were including the site style step as well.

I just had a personal revelation that [Blog](https://a8cvm2.wordpress.com/) uses `pub/modern-business` and [Professional](https://a8cvm4.wordpress.com/) uses `pub/elegant-business`... so I'm assuming it's all fine and dandy, like polka-dot pants. 🐆